### ISSUE TITLE
is there anyway to increase the wave zoom on the top to be at least 10 minutes instead of 20 second

### ISSUE THREAD

**Issue by reaganscofield** [bbc/peaks.js#225]
Good Day! please is there any way to decrease the zoom on the top of the waveform because its goes until 20 seconds and pass to another zoom but i want it to be 10 minutes instead of 20 seconds

**Comment by chrisn** [comment]
Great question. At the moment, there isn't a way to express this directly to Peak.js, so you'd have to calculate the appropriate zoom levels based on the image width, duration you want to show, and audio sample rate.

The `zoomLevels` option to `Peaks.init()` sets the fixed zoom levels for the zoomable waveform view. These are expressed in audio samples per pixel, with defaults of 512, 1024, 2048, 4096.

The initial zoom level is the first of these, so 512 samples/pixel.

This example shows how to use zoom levels of 5, 10, 20, 40 minutes. The `setZoom(1)` call selects 10 minutes as the initial zoom level.

```javascript
var minutes = 5;
var secondsPerMinute = 60;
var audioSampleRate = 48000; // Hz
var imageWidth = 1000;

var baseZoom = minutes * secondsPerMinute * audioSampleRate / imageWidth;

var peaksInstance = Peaks.init({
    // other options as needed...
    zoomLevels: [baseZoom, baseZoom * 2, baseZoom * 4, baseZoom * 8],
});

peaksInstance.zoom.setZoom(1); // Select zoomLevels[1], i.e., 10 minutes
```

I'll consider a new API to make this easier.

**Comment by reaganscofield** [comment]
Thank you so much this was helpful to me!! and i don't know in the future if you can method to call next clips, would be fine

**Comment by chrisn** [comment]
This is resolved in #305.
